reactioncommerce/cli
# (feat) add create-plugin api command

Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

Adds a new command `create-plugin api <myplug-name>` which will clone the example
plugin repo into the `custom-packages` directory, remove the .git file and init .git so this can be checked into their own repo.

**To test:**

(pull and reinstall if running globally)

1. Install or run locally running `./index.js` (when you move around change the relative path)
2. run `reaction create-project api myserver` if you don't have a project already
3. move into the myserver directory
4. run `reaction create-plugin api myplugin`
5. Observe that there is now a plugin in the `custom-packages` directory with the name `myplugin` that has no git commit history because it is freshly init'd.
